src/components/Species/AddSpeciesModal.tsx

```diff
@@ -57,6 +65,18 @@ export default function AddSpeciesModal(props: AddSpeciesModalProps): JSX.Elemen
   const debouncedSearchTerm = useDebounce(record.scientificName, 250);
   const [showWarning, setShowWarning] = useState(false);
 
+  const [tooltipLearnMoreModalOpen, setTooltipLearnMoreModalOpen] = useState(false);
+  const [tooltipLearnMoreModalContent, setTooltipLearnMoreModalContent] = useState<
+    TooltipLearnMoreModalData | undefined
+  >(undefined);
+  const openTooltipLearnMoreModal = ({ content, title }: TooltipLearnMoreModalData) => {
+    setTooltipLearnMoreModalContent({ content, title });
```

Comment:
why destructure `TooltipLearnMoreModalData` here? could you not simply pass through the data?

Reply:
I'm not sure why I decided to destructure the data here 🤔 Because you're right, I could simply pass it through. Will update.